crenata/discord/embed/meal.py

```diff
@@ -0,0 +1,51 @@
+from typing import Any, Literal
+
+from crenata.abc.builder import AbstractEmbedBuilder
+from crenata.utils.datetime import datetime_to_readable, to_datetime
+
+
+class MealEmbedBuilder(AbstractEmbedBuilder):
```

Comment:
```suggestion
class MealEmbedBuilder(AbstractEmbedBuilder):
    def __init__(self) -> None:
        super().__init__()

```
`AbstractEmbedBuilder`의 `__init__` 호출